docs/tutorial/part-four/index.md

```diff
@@ -50,7 +50,22 @@ and APIs of all sorts.
 **Gatsby's data layer lets you pull data from these (and any other source)
 directly into your components**—in the shape and form you want.
 
-## How Gatsby's data layer uses GraphQL to pull data into components
+## Using Unstructured Data vs GraphQL
```

Comment:
I'm not sure "Using Unstructured Data" as a top-level heading is a good phrase in the tutorial. I'd say maybe nix this, and just leave the intro to this with "Do I have to use GraphQL and source plugins to pull data into Gatsby sites?"

The phrase "unstructured data" makes more sense after this question